docs/meshstack.cloudfoundry.index.md

```diff
@@ -3,62 +3,49 @@ id: meshstack.cloudfoundry.index
 
-For Cloud Foundry, meshStack provides org and space creation and configuration, user management and SSO via Cloud Foundry's UAA.
+meshStack supports project creation, configuration, access control, quota management and billing for Cloud Foundry.
 
 ## Integration Overview
 
-> The recommended way to set up Cloud Foundry as a meshPlatform is via the public terraform [Cloud Foundry meshPlatform Module](https://github.com/meshcloud/terraform-cloudfoundry-meshplatform).
-
-To enable integration with Cloud Foundry, Platform Operators deploy and configure the meshStack Cloud Foundry connector to make Cloud Foundry platforms available at their meshStack instance.
+To enable integration with Cloud Foundry, Platform Operators configure one or multiple `meshPlatform`s of `PlatformType` Cloud Foundry in the [Platform Administration](./administration.platforms.md) in meshPanel.
 
-meshStack provides users access to Cloud Foundry (CF) instances via the OIDC protocol for authentication while it replicates permission rights directly to authorize correct access.
+## Prerequisites 
 
-When accessing the platform the user has an OIDC Token (JWT) which is issued by the meshIdB (and potentially by an upstream corporate identity provider, cf. [Identity Federation](meshstack.identity-federation.md)). Cloud Foundry's Auth component UAA validates the token upon access. Also for CF access within meshPanel the token is used to request status information about apps and services to display within meshPanel.
+Your Cloud Foundry must support the v3 Cloud Foundry API.
+We have officially validated meshStack with support for Cloud Foundry `v3.108.0`.
 
-The tenant [replication](./meshcloud.tenant.md) ensures spaces and orgs are created within the CF platform and appropriate permission rights are set when users access the CF platform. If a user's meshProject permissions are modified, meshStack updates the permissions for this user accordingly within the CF platform.
+### UAA Federated Users
 
-![Cloud Foundry Architecture](assets/cf-architecture.svg)
+meshStack will identify and assign users to roles in Cloud Foundry based on their `euid` (external user id) as described in [Identity Federation](meshstack.identity-federation.md#externally-provisioned-identities).
 
-## Cloud Foundry Versions
+You should set up your Cloud Foundry and UAA UAA so that they store the `euid` value in the `User.userName` field of the [UAA User object](https://docs.cloudfoundry.org/api/uaa/version/77.8.0/index.html#get) and `User.username` field of the [Cloud Foundry user object](http://v3-apidocs.cloudfoundry.org/version/3.164.0/index.html#users).
 
-meshcloud officially validates compatibility with the following API versions:
+## Integration Configuration
 
-* `2.155.0`
-* `2.181.0`
+> The recommended way to set up Cloud Foundry as a meshPlatform is via the public terraform [Cloud Foundry meshPlatform Module](https://github.com/meshcloud/terraform-cloudfoundry-meshplatform).
```

Comment:
```suggestion
> The recommended way to integrate Cloud Foundry as a platform in meshStack is via the public terraform [Cloud Foundry Platform Module](https://github.com/meshcloud/terraform-cloudfoundry-meshplatform).
```

Hmm now we did call the repo meshPlatform in the end. I leave it up to you whether you want to keep the meshPlatform naming but we did kind of decide to get rid of the meshNaming in our product.

Reply:
@Jelledb all of the platform integration tf modules are called `terraform-$provider-meshplatform`. The prefix `terraform-$provider` is mandatory by terraform registry conventions. If you want to call them differently, we'd need to migrate/rename all of them. If you want to get rid of that naming everywhere, that would require a dedicated backlog item.